main.py

```diff
@@ -246,26 +252,44 @@ def match_flows(self, switch, args, stored_flows, many=True):
             return None
         return response
 
-    # pylint: disable=redefined-outer-name
-    # pylint: disable=too-many-branches
+    def get_action(self, switch, table_id, args, stored_flows, actions):
```

Comment:
The name of this method doesn't reflect what it does, it's getting actions and not just a single action, and it's also resolving the table context.